t/t2407-sdbus.t

```diff
@@ -0,0 +1,117 @@
+#!/bin/sh
+# ci=system
+
+test_description='Test flux systemd sd-bus gatweay'
+
+. $(dirname $0)/sharness.sh
+
+if ! flux version | grep systemd; then
+	skip_all="flux was not built with systemd"
+	test_done
+fi
+if ! systemctl --user show --property Version; then
+	skip_all="user systemd is not running"
+	test_done
+fi
```

Comment:
similar to my comment in the unit tests, should there be additional checks?  I imagine common scenario is user running tests and this might be run against a user service which may not be setup.